### PR TITLE
test(hx-password-control): component tests surf-2056

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

Test Case `<hx-password-control>`

image

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2056

## Before you request a review for this PR:

- [x] Did you add component tests for any new code?
- [x] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [x] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/elements/hx-password-control/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        });
+    });
+
+    describe('test Control Element', () => {
+        it('to test input[type="password"]', async () => {
+            const elSelector = 'hx-password-control > input#pwdDemo';
+            const id = 'pwdDemo';
+            const component = /** @type { HXPasswordElement } */ await fixture(mockup);
+            const query = component.querySelector(elSelector);
+            const queryId = query.id;
```
Comment (lalithkarikelli) [inline src/elements/hx-password-control/index.spec.js:66]: Please change this test as per discussion , write test case to check control element.

`const query = fragment.controlElement;`

File: src/elements/hx-password-control/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+            expect(shadow).to.be.null;
+        });
+    });
+
+    describe('test Control Element', () => {
+        it('to test input[type="password"]', async () => {
+            const elSelector = 'hx-password-control > input#pwdDemo';
+            const id = 'pwdDemo';
+            const component = /** @type { HXPasswordElement } */ await fixture(mockup);
```
Comment (lalithkarikelli) [inline src/elements/hx-password-control/index.spec.js:64]: change variable component to fragment.

`const fragment = /** @type { HXPasswordElement } */ await fixture(mockup);`

File: src/elements/hx-password-control/index.spec.js (hunk 1 above)
Comment (BY00565233) [reply src/elements/hx-password-control/index.spec.js]: Changes have been done, Please review

File: src/elements/hx-password-control/index.spec.js (hunk 2 above)
Comment (BY00565233) [reply src/elements/hx-password-control/index.spec.js]: Changes have been done, Please review

**Pull request merged**